### DISCUSSION TITLE
How to exclude some routes from multi tenant scope

### DISCUSSION THREAD

**Discussion by usmanibrahim74** [filamentphp/filament#13392] [Help]
### Package

Panel builder

### Package Version

v3

### How can we help you?

I'm using multi tenant option. I have some custom api routes. when I try to access them it gives me error saying unauthenticated. I guess it is thinking the route belongs to a tenant because similar routes exists in filament app. is there a way to exclude all api routes from multi tenant scope? there is trick I can do is changing the routes to a different path so that filament don't consider them as a tenant route. But it would be great if there could be a better solution.

**Comment by leandrocfe** [comment]
Are you using api routes?
https://laravel.com/docs/11.x/routing#api-routes
